### PR TITLE
Replace broken VS Code link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Beagall**
# PR Summary

Replace a broken link to Visual Studio Code in the CONTRIBUTING.md file with a working one.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ To run the link-checker, follow these steps:
 
 ### Code Editor
 
-You should use the multi-platform [Visual Studio Code (VS Code)][use-vscode-editor].
+You should use the multi-platform [Visual Studio Code (VS Code)](https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/using-vscode?view=powershell-6).
```
Comment (iSazonov) [inline .github/CONTRIBUTING.md]: Please find "use-vscode-editor" below in the file and correct the link there.

**Comment by stevend811** [comment]
@iSazonov I corrected the link in the bottom of the file.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -395,7 +395,7 @@ Once you sign a CLA, all your existing and future pull requests will have the st
 [up-for-grabs]: https://github.com/powershell/powershell/issues?q=is%3Aopen+is%3Aissue+label%3AUp-for-Grabs
 [semantic linefeeds]: https://rhodesmill.org/brandon/2012/one-sentence-per-line/
 [PowerShell-Docs]: https://github.com/powershell/powershell-docs/
-[use-vscode-editor]: ../docs/learning-powershell/using-vscode.md#editing-with-visual-studio-code
+[use-vscode-editor]: https://docs.microsoft.com/powershell/scripting/components/vscode/using-vscode?view=powershell-6
```
Comment (iSazonov) [inline .github/CONTRIBUTING.md]: @stevend811 The docs is for PowerShell _script_ development. We need link to using VS Code for C# development.

Comment (stevend811) [reply .github/CONTRIBUTING.md]: @iSazonov All right, I had chosen that link based on the broken link previously leading to a [page on script development](https://github.com/PowerShell/PowerShell/blob/7076d806314e12a1b9ddef2a90d6108ddb7f50b6/docs/learning-powershell/using-vscode.md). Would https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio-code be a good choice here?

Comment (iSazonov) [reply .github/CONTRIBUTING.md]: The new link looks more appropriated.

**Comment by stevend811** [comment]
@iSazonov Ready for another review.

**Review by iSazonov — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]
The link used in this PR is more appropriate as the article is about contributing to PowerShell code (CSharp).

**Comment by iSazonov** [comment]
@stevend811 Thanks for your contribution!

**Pull request merged**